src/utils/interfaces/data/logs.ts

```diff
@@ -38,7 +38,7 @@ export interface AdvancedFormInitialValues extends SimpleFormInitialValues {
   end_air?: number;
   air_type?: string;
   date_dived?: string;
-  date_posted?: string;
```

Comment:
We don't set this on our end, so I left it as optional so the compiler does not bleat.

Reply:
i think in this case it's okay because it should always exist otherwise it's an issue from the API and not the app. the contract with the server is that the client should be able to expect this field

Reply:
i see, that makes sense. the reason i did it here was that the compiler complained that you can't use `new Date()` with a null value. I guess the other option would've been to affirm that it exists inside the `new Date()` function

Reply:
Yeah, using a ternary would eliminate the warning, especially when you're sure that value exists.